package.json

```diff
@@ -44,6 +45,9 @@
       "<rootDir>/integration-tests/",
       "<rootDir>/packages/"
     ],
+    "setupFiles": [
+      "raf/polyfill"
```

Comment:
React 16 requires a `requestAnimationFrame` polyfill for tests to run properly.

Reply:
Looks like Jest `21.3.x` included the polyfill by default. Maybe we can bump the version of Jest here (or a separate PR?) https://github.com/facebook/jest/pull/4568#issuecomment-336562985